### ISSUE TITLE
Can't resolve 'react-measure/lib/with-content-rect'

### ISSUE THREAD

**Issue by huiliu1110** [juliankrispel/react-text-selection-popover#18]
@juliankrispel Very nice component, but keep getting an error 
```
./node_modules/react-text-selection-popover/dist/index.es.js
Module not found: Can't resolve 'react-measure/lib/with-content-rect' in '/node_modules/react-text-selection-popover/dist'
``` 

should we take a look on [this line](https://github.com/juliankrispel/react-text-selection-popover/blob/master/src/index.js#L4)?

**Comment by juliankrispel** [comment]
hey @haleyhuiliu let me have a quick look

**Comment by huiliu1110** [comment]
Hello @juliankrispel  any update on this issue? did you observe the same issue on your end?

**Comment by jvalente** [comment]
Hello folks, it seems like the `withContentRect` import is a bit dated.

It should be: `import { withContentRect } from "react-measure";`

Submitted a fix [here](https://github.com/juliankrispel/react-text-selection-popover/pull/19).

Cheers!

**Comment by AdrianTudC** [comment]
@juliankrispel can you approve @jvalente 's commit? The library is unusable at the moment.

**Comment by juliankrispel** [comment]
this is now fixed in version `2.1.3`, please try again :+1:
